cmd/acctestgen/main.go

```diff
@@ -0,0 +1,287 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func main() {
+	var dryrun = flag.Bool("dryrun", false, "preview list of files to be created")
+	flag.Parse()
+
+	service := flag.Arg(0)
+	service = strings.ToLower(service)
+	err := validate(service)
+	if err != nil {
+		fmt.Printf("\nERROR: %s\n\n", err)
+		os.Exit(1)
+	}
+
+	datasourcePath := fmt.Sprintf("internal/services/%s/data_source_test.go", service)
+	resourcePath := fmt.Sprintf("internal/services/%s/resource_test.go", service)
+	testdataDirPath := fmt.Sprintf("internal/services/%s/testdata/", service)
+	resourceDataPath := fmt.Sprintf("internal/services/%s/testdata/resourcebasicconfig.tf", service)
+	datasourceDataPath := fmt.Sprintf("internal/services/%s/testdata/datasourcebasicconfig.tf", service)
```

Comment:
we can probably refine the names here. given they are packaged now whereas the migration needed to keep the names for the purposes of tracking.

```suggestion
	resourceDataPath := fmt.Sprintf("internal/services/%s/testdata/basic.tf", service)
	datasourceDataPath := fmt.Sprintf("internal/services/%s/testdata/datasource_basic.tf", service)
```